src/foam/u2/grid/seutil.js

```diff
@@ -0,0 +1,396 @@
+console.log("Loading SE.util");
```

Comment:
I wasn't aware of foam.LIB(), that's amazing!
What's the main difference between foam.LIB and foam.CLASS? 
Is there any other foam.XXX I should be aware of? 

Where should I put my util libs? src/foam/util, maybe? 
Should I break up my Util into  smaller foam.LIB libs?